packages/pymedphys_gamma/tests/gamma/test_gamma_shell.py

```diff
@@ -28,7 +28,8 @@
 import pytest
 
 import numpy as np
-from pymedphys_analysis.gamma import gamma_shell, calculate_coordinates_shell
+from pymedphys.gamma import gamma_shell
+from pymedphys_gamma.implementation.shell import calculate_coordinates_shell
```

Comment:
So it's okay for tests to import from subpackages like `pymedphys_gamma` as well as `pymedphys`?

Reply:
So, I made a decision here. This is in the tests only, not in the sub modules. This comes at a trade off. We can no longer test each sub module apart from the main pymedphys package (quite a big downside). However, I want to have any API that the wep app is using be directly used within the tests... (big plus).

Submodules themselves cannot import from pymedphys. Tests though (now) can.